README.md

```diff
@@ -1,38 +1,43 @@
+## 🎨 Welcome to the AsyncAPI brand
+In this project, we focus on all things related to the creative development of the AsyncAPI brand. The overarching goal for this repo is to provide a space where we can collaborate to define visuals such as logos, colors, typography, iconography, and more! All are welcome to contribute ideas and work where you feel comfortable.
 
-## Ready to contribute? 🙌 💡
+Before you submit your contribution, please read our [Code of Conduct](https://github.com/asyncapi/.github/blob/master/CODE_OF_CONDUCT.md) and [Contribution Guidelines](https://github.com/asyncapi/asyncapi/blob/master/CONTRIBUTING.md#contributing-to-asyncapi).
+
+## ✨ Brand Guidelines
+Navigate through our brand guidelines using the links below
+
+- [Logo](brand-guidelines/logo/README.md)
+- [Color](brand-guidelines/color/README.md)
+- [Typography](brand-guidelines/typography/README.md)
+- [Branded Tools](brand-guidelines/branded-tools/README.md)
+
+## 🙌 💡 Ready to contribute?
 There are plenty of ways to share your contribution with this project, and you don't have to be a designer to do so! We welcome feedback and work alike in the following areas:
 
-- Design System
 - Logo design/usage
 - Brand identity
 - Typography
 - Brand guidelines
 - Iconography
 - Mascot ideation/illustration
-- UI Kit
 - Marketing/Social Media design
 
 Issues in this repo are open to all who would like to give feedback and/or provide visual work.
 
-## Roadmap 🚗 💨
+## 🚗 💨 Roadmap
 View the chart below to learn about the current goals for the AsyncAPI brand!
 
-|  Step  |                   Goal                     |
+|   Status     |                   Goal                     |
 | :----: | ------------------------------------------ |
-|   1    | Refresh logos, colors, typography, and establish voice/tone |
-|   2    | Create diverse set of iconography to complement tooling, website, and visual marketing |
-|   3    | Brand guidelines website |
-|   4    | Ideate and illustrate mascot to complement new brand style |
-|   5    | Design marketing campaign for brand roll-out |
-|   6    | Develop design system and implement across products/websites |
-|   7    | Create UI Kit - classes and react components |
-|   8    | New website design based on design system and UI Kit |
-|   9    | Feedback and iteration on design system and UI Kit |
+|   In progress    | Brand guidelines |
```

Comment:
You may want to link to a Github project instead of maintaining this here. In my experience, it's easy to forget to update it.

Reply:
Good idea! I started exploring this a little bit in the past and haven't really kept up with it. I will definitely re-explore how the project can replace this chart!

Reply:
😍 Love it!